taxcalc/tests/test_pufcsv.py

```diff
@@ -239,3 +239,42 @@ def test_mtr(tests_path, puf_path):  # pylint: disable=redefined-outer-name
         msg += '---            and rerun test.                ---\n'
         msg += '-------------------------------------------------\n'
         raise ValueError(msg)
+
+
+@pytest.mark.requires_pufcsv
+def test_credit_reforms(puf_path):  # pylint: disable=redefined-outer-name
```

Comment:
Some of both: the test of the personal nonrefundable credit is **not** in the `comparison/reform_results.py` suite.